### ISSUE TITLE
Camera support

### ISSUE THREAD

**Issue by claymca72** [damianxd/homebridge-vera#191]
Is there support for cameras?

**Comment by tsfischer** [comment]
Unfortunately I think this plugin is currently abandonware... the original author as well as at other major contributors have moved on and are not using Vera anymore. The last release was in 2018... I'm currently running a version I patched to work with the latest homebridge... I had the PR approved but a release has not happened. I am not fluent in Node.js so I don't feel I can take this project over, although I'm happy to contribute if someone else does.

Looking through the other Vera plugin options, I'm not sure there is another one that's in significantly better health.

**Comment by damianxd** [comment]
You are right, at least on my case I've moved to Homey and I'm developing for it at the moment. I could give the access to npm to someone else who wants to take over, but for the moment, tell me if your pull request isn't on npm, I could push the release there when I'm done with my regular work.

**Comment by drewcovi** [comment]
Same story here. It was a good run with Vera, but as we can’t get homey in the states, it was Hubitat all the way for me.

**Comment by tsfischer** [comment]
As an active user and professional software engineer I would love to help out. However I have zero experience with node, NPM, or any of the tech stack that Homebridge PIs are based on, other than the tiny bit I've hacked away at for my own use.

I would fully support and contribute toward anyone who would take this effort over, I just don't feel I'm the right guy for the job.

**Comment by drewcovi** [comment]
We all start somewhere. 🙂

Happy to walk through my process for working on the plug-in locally in an isolated and separate “home” in iOS.

**Comment by claymca72** [comment]
Thanks

**Comment by tsfischer** [comment]
@drewcovi I know it's nearly a year later but could you comment on your process of working not he plugin locally that you mentioned above? I'm going to stop short of committing to be an active contributor here, but I'd like to learn a bit more about how this all works because I'm not all that comfortable having my home automation rely on something that nobody (including me) is supporting. And this plugin works really well for me right now, and there doesn't seem to be anything else suitable out there for a Vera to Homebridge plugin. 

I'd also love to know what's at Master that hasn't been released yet, and possibly should be. Basically try to breathe som life back into this project, even though I'm not a Node guy...

Thanks!

**Comment by tsfischer** [comment]
I switched my 'production' HomeBridge to running the tip of `master` for the last few days and it seems to be working fine. I also made a new PR to eliminate the additional warnings that were brought up in my last PR. 

Just wondering what next steps are to get the PRs merged and possibly a new release. This is something I'm willing to help with if given access.
